script.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python3
+
+import psycopg2
+import requests
+from bs4 import BeautifulSoup
+
+
+def connect():
+    conn = psycopg2.connect(host='db', user="postgres", database="postgres-2", password='mmuntean1!')
+    cur = conn.cursor()
+    cur.execute("SELECT name, description, imdb_id FROM public.movies_movie WHERE movies_movie.description='';")
```

Comment:
It might be worth adding a limit to the SQL query to avoid processing too many rows at a time.
For debugging purposes it is fine, but if we use only the imdb_id column later, it's worth selecting only that column.